### DISCUSSION TITLE
Is it possible to set a static Static IP?

### DISCUSSION THREAD

**Discussion by bradkeller** [gtxaspec/wz_mini_hacks#596] [Q&A]
Is there a way to se a static IP address without a DHCP reservation?

**Comment by virmaior** [comment]
you're going to need to a bit of spelunking at `https://github.com/gtxaspec/wz_mini_hacks/blob/master/SD_ROOT/wz_mini/etc/rc.d/S03ethernet` or thereabouts

you'll need to run `ifconfig` and set the static IP when you initialize it.

basic syntax is:
````
ifconfig eth0 192.168.1.2 netmask 255.255.255.0;
route add default gw 192.168.1.1 eth0;
````

but you're going to need make sure you're doing it to the right interface (here eth0 , not sure what the correct one would be)
